django/http/multipartparser.py

```diff
@@ -67,17 +68,21 @@ def __init__(self, META, input_data, upload_handlers, encoding=None):
         """
         # Content-Type should contain multipart and the boundary information.
         content_type = META.get("CONTENT_TYPE", "")
-        if not content_type.startswith("multipart/"):
-            raise MultiPartParserError("Invalid Content-Type: %s" % content_type)
 
-        # Parse the header to get the boundary to split the parts.
         try:
-            ctypes, opts = parse_header(content_type.encode("ascii"))
+            content_type.encode("ascii")
         except UnicodeEncodeError:
             raise MultiPartParserError(
                 "Invalid non-ASCII Content-Type in multipart: %s"
                 % force_str(content_type)
             )
+
+        if not content_type.startswith("multipart/"):
+            raise MultiPartParserError("Invalid Content-Type: %s" % content_type)
```

Comment:
Is there a reason to move this check? I'd move it to the previous place.

Reply:
Thanks for mentioning that. I'll move it to it's previous place.